main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/prometheus/alertmanager/template"
+	"log"
+	"net/http"
+	"os"
+)
+
+type JsonResponse struct {
```

Comment:
My go-lint tells me 
- "exported type JsonResponse should have comment or be unexported". 
- "type JsonResponse should be JSONResponse"
I agree.
```suggestion
// Webhook http response
type JSONResponse struct {
```